mailbox/api/src/main/java/org/apache/james/mailbox/MessageManager.java

```diff
@@ -74,8 +75,9 @@
 
     /**
      * Return if the Mailbox is writable
+     *
      * @deprecated use
-     *             {@link #getMetaData(boolean, MailboxSession, org.apache.james.mailbox.MessageManager.MetaData.FetchGroup)}
+     * {@link #getMetaData(boolean, MailboxSession, org.apache.james.mailbox.MessageManager.MetaData.FetchGroup)}
```

Comment:
all these javadoc changes are not bad, but are they also related to this PR?

Reply:
when moving Mailbox from store to api. I reformat the MessageManager class.